pkg/apply/destroyer.go

```diff
@@ -42,20 +43,22 @@ func NewDestroyer(factory cmdutil.Factory, invClient inventory.InventoryClient)
 		return nil, err
 	}
 	return &Destroyer{
-		pruner:       pruner,
-		StatusPoller: statusPoller,
-		factory:      factory,
-		invClient:    invClient,
+		pruner:           pruner,
+		StatusPoller:     statusPoller,
+		factory:          factory,
+		invClient:        invClient,
+		WrapInventoryObj: inventory.WrapInventoryInfoObj,
```

Comment:
This property is private in the Applier and public in the Destroyer. Is there a reason why it isn't private in both?